lib/Notes/NotesSmartAccordion/tests/notes-smart-accordion-test.js

```diff
@@ -99,7 +99,7 @@ describe('Notes smart accordion', () => {
         await notesModal.notes(0).clickCheckbox();
       });
 
-      it('should not allow to unassign', () => {
+      it.only('should not allow to unassign', () => {
```

Comment:
Oops!

Reply:
Yep. * long-winded-sigh....

Reply:
I guess if es-lint doesn't catch it, sonar cloud will at least make us look...